### PR TITLE
added html and css for UI add doctor

### PR DESCRIPTION
**Pull request by baciu-radu**

### CODE REVIEW

File: src/main/resources/static/assets/css/style-add-doctor.css [hunk 1]
```diff
@@ -0,0 +1,2 @@
...delay:.6s;animation-delay:.6s}.animated.delay-07s{-webkit-animation-delay:.7s;animation-delay:.7s}.animated.delay-08s{-webkit-animation-delay:.8s;animation-delay:.8s}.animated.delay-09s{-webkit-animation-delay:.9s;animation-delay:.9s}.rounded-20{border-radius:20px!important}.rounded-500{border-radius:500px!important}.btn-close{background:none;box-shadow:none!important;font-size:20px;font-weight:500;line-height:1;opacity:1;outline:none!important;text-shadow:none;transition:color .2s ease-in-out}.btn-close:hover{color:#ed5564}.map{height:100%}.icons-list{display:flex;flex-wrap:wrap;margin:-.5rem}.icons-list .icon{font-size:30px;margin:.5rem}.add-action-box{bottom:5.71429rem;position:fixed;right:1.42857rem;z-index:9997}.add-action-box .btn{font-size:30px;min-width:5rem!important}.department .team{flex-wrap:wrap}.department .team .team-img{box-shadow:0 18px 24px rgba(0,0,0,.12);margin-right:-10px}.bg-gradient{background:linear-gradient(90deg,#fff,#ebebeb)!important}.address-col{min-width:150px}.fs-20{font-size:20px}.fs-30{font-size:30px}.fs-48{font-size:48px}.personal-info-card .card-body{padding-top:0!important}.personal-info-card .user-actions{margin-top:-50px}.opacity-50{opacity:.5}
```
Comment (andreimladin) [inline src/main/resources/static/assets/css/style-add-doctor.css]: please rename it to a more general css and override there only the id or class that you want to change

**Review by AlinaSastras — DISMISSED** [review]

**Review by adrianlibenciuc — DISMISSED** [review]

File: src/main/resources/static/assets/css/style2.css [hunk 2]
```diff
@@ -1,2 +0,0 @@
...delay:.6s;animation-delay:.6s}.animated.delay-07s{-webkit-animation-delay:.7s;animation-delay:.7s}.animated.delay-08s{-webkit-animation-delay:.8s;animation-delay:.8s}.animated.delay-09s{-webkit-animation-delay:.9s;animation-delay:.9s}.rounded-20{border-radius:20px!important}.rounded-500{border-radius:500px!important}.btn-close{background:none;box-shadow:none!important;font-size:20px;font-weight:500;line-height:1;opacity:1;outline:none!important;text-shadow:none;transition:color .2s ease-in-out}.btn-close:hover{color:#ed5564}.map{height:100%}.icons-list{display:flex;flex-wrap:wrap;margin:-.5rem}.icons-list .icon{font-size:30px;margin:.5rem}.add-action-box{bottom:5.71429rem;position:fixed;right:1.42857rem;z-index:9997}.add-action-box .btn{font-size:30px;min-width:5rem!important}.department .team{flex-wrap:wrap}.department .team .team-img{box-shadow:0 18px 24px rgba(0,0,0,.12);margin-right:-10px}.bg-gradient{background:linear-gradient(90deg,#fff,#ebebeb)!important}.address-col{min-width:150px}.fs-20{font-size:20px}.fs-30{font-size:30px}.fs-48{font-size:48px}.personal-info-card .card-body{padding-top:0!important}.personal-info-card .user-actions{margin-top:-50px}.opacity-50{opacity:.5}
```
Comment (andreimladin) [inline src/main/resources/static/assets/css/style2.css:1]: add only the style that you changed, not that entire file
+ avoid naming files 1 2 or random numbers without any sense, change to a more suggestive name like - icarenow-style.css

File: src/main/resources/templates/icn-login.html [hunk 3]
```diff
@@ -37,7 +37,7 @@
 <!-- .main-loader -->
 
 <div class="page-box">
-    <div class="app-container page-sign-in">
+    <div class="app-container page-icn-login">
```
Comment (andreimladin) [inline src/main/resources/templates/icn-login.html:40]: where is the new css file referred?

File: src/main/resources/templates/icn-add-doctor.html [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    <link rel="stylesheet" href="../assets/css/simple-line-icons.css">
+    <link rel="stylesheet" href="../assets/css/jquery.typeahead.css">
+    <link rel="stylesheet" href="../assets/css/datatables.min.css">
+    <link rel="stylesheet" href="../assets/css/bootstrap-select.min.css">
+    <link rel="stylesheet" href="../assets/css/Chart.min.css">
+    <link rel="stylesheet" href="../assets/css/morris.css">
+    <link rel="stylesheet" href="../assets/css/leaflet.css">
+
+    <!-- Theme CSS -->
+    <link rel="stylesheet" href="../assets/css/style.css">
```
Comment (andreimladin) [inline src/main/resources/templates/icn-add-doctor.html]: where is the new css file referred?

File: src/main/resources/templates/icn-signup.html [hunk 5]
```diff
@@ -36,7 +36,7 @@
 <!-- .main-loader -->
 
 <div class="page-box">
-    <div class="app-container page-sign-up">
+    <div class="app-container page-icn-signup">
```
Comment (andreimladin) [inline src/main/resources/templates/icn-signup.html:39]: where is the new css file referred?

**Review by andreimladin — CHANGES_REQUESTED** [review]

File: src/main/resources/static/assets/css/style2.css (hunk 2 above)
Comment (baciu-radu) [reply src/main/resources/static/assets/css/style2.css:1]: Eu L-am scos pe asta. Cu Git rm

Comment (baciu-radu) [reply src/main/resources/static/assets/css/style2.css:1]: A rămas doar style.CSS

File: pom.xml [hunk 6]
```diff
@@ -41,11 +41,11 @@
             <scope>runtime</scope>
             <version>2.1.212</version>
         </dependency>
-        <dependency>
-            <groupId>org.modelmapper</groupId>
-            <artifactId>modelmapper</artifactId>
-            <version>3.1.0</version>
-        </dependency>
+<!--        <dependency>-->
```
Comment (andreimladin) [inline pom.xml]: to be removed

File: src/main/java/us/icarenow/web/controller/AdminController.java [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+    public ModelAndView addDoctorAction(@Valid @ModelAttribute("doctorForm") DoctorForm doctorForm, BindingResult result, ModelMap model) {
+        if (result.hasErrors()) {
+            return new ModelAndView("icn-doctors");
+        }
+
+        System.out.println(doctorForm.toString());
+        adminService.addDoctor(doctorForm);
+
+        List<Doctor> doctors = doctorService.getAllUsers();
+        List<User> users = userService.getAllUsers();
```
Comment (andreimladin) [inline src/main/java/us/icarenow/web/controller/AdminController.java:59]: n x m -> Replace @OneToMany on Doctor

File: src/main/java/us/icarenow/web/controller/AdminController.java [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+        });
+
+        ModelAndView doctorListMV = new ModelAndView("icn-doctors", "doctorForm", new DoctorForm());
+        doctorListMV.addObject("doctorList", doctorUserDTOS);
+        return doctorListMV;
+    }
+
+    @GetMapping("/doctors")
+    public ModelAndView addDoctorView() {
+        List<Doctor> doctors = doctorService.getAllUsers();
```
Comment (andreimladin) [inline src/main/java/us/icarenow/web/controller/AdminController.java:89]: The same for user on doctor (OneToMany)
ManyToMany between doctor and specility

File: src/main/java/us/icarenow/web/repository/DoctorRepository.java [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+import org.springframework.stereotype.Repository;
+import us.icarenow.web.model.entity.Doctor;
+
+import java.util.List;
+
+@Repository
+public interface DoctorRepository extends JpaRepository<Doctor, Long> {
+
+    @Override
+    List<Doctor> findAll();
```
Comment (andreimladin) [inline src/main/java/us/icarenow/web/repository/DoctorRepository.java:13]: it's not needed

File: src/main/java/us/icarenow/web/service/DoctorService.java [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+
+    @Autowired
+    private SpecialtyRepository specialtyRepository;
+
+    @Autowired
+    ModelMapper modelMapper;
+
+
+    public void addDoctor(DoctorForm doctorForm, long userId) {
+
```
Comment (andreimladin) [inline src/main/java/us/icarenow/web/service/DoctorService.java:31]: empty

File: src/main/java/us/icarenow/web/service/DoctorService.java [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+       Doctor doctor = new Doctor(userId, doctorForm.getFirstName(), doctorForm.getLastName());
```
Comment (andreimladin) [inline src/main/java/us/icarenow/web/service/DoctorService.java:32]: you should have received a specialtyId from the form

File: src/main/java/us/icarenow/web/service/InsuredPatient.java [hunk 12]
```diff
@@ -0,0 +1,25 @@
+package us.icarenow.web.service;
+
+import org.springframework.stereotype.Service;
+
+import java.io.Serializable;
+
+@Service
+public class InsuredPatient implements Serializable {
```
Comment (andreimladin) [inline src/main/java/us/icarenow/web/service/InsuredPatient.java:8]: actually is a util class

**Review by andreimladin — APPROVED** [review]

**Review by adrianlibenciuc — APPROVED** [review]

**Pull request merged**